src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -220,3 +220,66 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('subnet', help='Name or ID of subnet. If name is supplied, `--vnet-name` must be supplied.')
         c.argument('vnet_name', help='Name of a virtual network.')
         c.argument('ip_address', help='IPv4 address or CIDR range.')
+
+    with self.argument_context('acr scope-map') as c:
```

Comment:
do you want to add `name` here as well? Seems like its needed everywhere under this group.

```
c.argument('scope_map_name', options_list=['--name', '-n'], help='The name of the target scope map.', required=True)
```

Reply:
Actually we have the command `az acr scope-map list`, which is not shown here as we just need the registry name which is already declared. Because of it we can't move this argument up